front-end-admin/src/components/user/UserList.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="jumbotron">
+    <div class="container">
+      <router-link class="btn btn-default" to="/user/register">Add User&raquo;</router-link>
+      <!-- TODO: Add form for searching through users -->
+      <h3>Users</h3>
+      <table class="table">
+        <thead>
+        <tr>
+          <th>User Name</th>
+          <th>Email</th>
+          <th>Display Name</th>
+          <th>Joined On</th>
+        </tr>
+        </thead>
+        <tbody id="fulluserTBody">
+        <tr v-for="user in users" :key="user.id">
+          <td>{{user.userName}}</td>
+          <td>{{user.email}}</td>
```

Comment:
You'll need to create <td> tags for the user's display name and their join date. If that's not provided in the back-end then no worries for now

Reply:
The table only has columns for user_id, user_name, user_email, user_password, and user_permissions, so we'll have to add the display name and join date to the table.